resources/rgbscripts/lines.js

```diff
@@ -35,17 +34,23 @@ var testAlgo;
     algo.linesSize = 10;
     algo.properties.push("name:linesSize|type:range|display:Size|values:1,32|write:setLinesSize|read:getLinesSize");
     algo.linesVariability = 0;
-    algo.properties.push("name:linesVariability|type:range|display:Size Variablity|values:0,100|write:setVariability|read:getVariability");
     algo.linesType = 0;
     algo.properties.push("name:linesType|type:list|display:Type|values:Horizontal,Vertical,Plus,X,Star,Left,Right,Up,Down,UpRight,UpLeft,DownRight,DownLeft|write:setType|read:getType");
     algo.linesBias = 0;
     algo.properties.push("name:linesBias|type:list|display:Starting Bias|values:Random,Top,Bottom,Left,Right,TopLeft,TopRight,BottomLeft,BottomRight|write:setBias|read:getBias");
-    algo.linesSlide = 0;
-    algo.properties.push("name:linesSlide|type:list|display:Slide|values:None,Up,Down,Left,Right|write:setSlide|read:getSlide");
```

Comment:
You removed "Slide" and "Rollover" so this is not backward compatible.
What about existing projects using those parameters?